share/spice/uv/uv.js

```diff
@@ -0,0 +1,57 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_uv = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error || !api_result.length || api_result.length !== 1) {
+            return Spice.failed('uv');
+        }
+
+        //extract city and date from first result
+        var result = api_result[0];
+        
+        var city = DDG.capitalizeWords(result.CITY.toLowerCase()),
+            state = result.STATE,
+            uvIndex = parseInt(result.UV_INDEX,10),
+            uvIndexName = uvIndex+"";
```

Comment:
You could update this var assignment to `uvIndexName = uvIndex >= 11 ? "11+" : uvIndex;`